libbpf-tools/readahead.bpf.c

```diff
@@ -34,8 +34,8 @@ int BPF_PROG(do_page_cache_ra)
 	return 0;
 }
 
-SEC("fexit/__page_cache_alloc")
-int BPF_PROG(page_cache_alloc_ret, gfp_t gfp, struct page *ret)
```

Comment:
This is NOT the right fix, it will breaks the tool on some kernels, please track down which kernel commit break the tool.

And this is NOT compilation error, please reword your commit message.

Reply:
Try again, error still here:

```
$ sudo ./readahead 
libbpf: failed to find kernel BTF type ID of '__page_cache_alloc': -3
libbpf: prog 'page_cache_alloc_ret': failed to prepare load attributes: -3
libbpf: prog 'page_cache_alloc_ret': failed to load: -3
libbpf: failed to load object 'readahead_bpf'
libbpf: failed to load BPF skeleton 'readahead_bpf': -3
failed to load BPF object
```

Reply:
FYI, this is how we handle kernel symbols renamed:
https://github.com/iovisor/bcc/blob/1a89e0c38eab9d4ea8bc0503b18386af8e604924/libbpf-tools/biotop.c#L395-L398

Reply:
Thanks, I'll learn it and try modify the code again